### PR TITLE
Update issue templates: convert "question" template to contact_links

### PR DESCRIPTION
**Pull request by masklinn**
Found that in the Rust project, replaces the "Get Started" button for Question (which opens a template containing the Discord link) by a direct link to the Discord server:
Screen Shot 2021-01-25 at 16 42 39
to
Screen Shot 2021-01-25 at 16 43 12

Also disables the "Open a blank issue" link in the footer.

Incidentally it looks like it might be possible to create a direct invite to a channel? 

I don't know if that only allows inviting people or if it can provide a link. In the latter case it could be better than linking to the server.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url: https://discord.gg/tokio
```
Comment (seanmonstar) [inline .github/ISSUE_TEMPLATE/config.yml:4]: The README includes a link directly to the channel: https://discord.gg/RFsPjyt

Comment (masklinn) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Excellent, I'll use that then!

Comment (masklinn) [reply .github/ISSUE_TEMPLATE/config.yml]: Done, seems to work fine.

**Review by seanmonstar — APPROVED** [review]
Thanks!

**Pull request merged**